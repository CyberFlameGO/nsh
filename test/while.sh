while_func() {
    while [ "$1" != "" ]; do
        echo $1
        shift
        echo next is $1
    done
}

while_func a b c d
