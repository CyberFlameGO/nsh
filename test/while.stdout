a
next is b
b
next is c
c
next is d
d
next is
